src/internal/item-container/index.tsx

```diff
@@ -253,6 +255,9 @@ function ItemContainerComponent(
 
     // Notify the respective droppable of the intention to insert the item in it.
     draggableApi.acquire(nextDroppable, childrenRef.current);
+    // setTransition(null);
```

Comment:
This one should be either removed or uncommented. I think uncommenting does make more sense